recipes/modern-cpp-kafka/all/test_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def requirements(self):
```

Comment:
Why do you need only for testing and it's not a dependency for modern-cpp-kafka?

Reply:
Oh, that's the tricky part -- we don't want to bind the `modern-cpp-kafka` (header-only API) to a specific `librdkafka` (the underlying C library) version. E.g. `modern-cpp-kafka/2022.06.15` could work together with `librdkafka/1.7.0`, `librdkafka/1.8.0`, `librdkafka/1.8.2` and `librdkafka/1.9.0 (not available in conan center yet)` , and user might want to pickup just a specific `librdkafka` version (not the one which `modern-cpp-kafka` depends on), -- some `librdkafka` version might have a few defects and users don't always want to pick up the latest one.

Then, the `connanfile.py` (for user's application) might be,
```
def requirements(self):
    self.requires("librdkafka/1.7.0")
    self.requires("modern-cpp-kakfa/2022.06.15")
```

Reply:
Okay, it sounds fine. We have other few projects which follow the same line.